stream_markers.go

```diff
@@ -70,3 +70,56 @@ func (c *Client) GetStreamMarkers(params *StreamMarkersParams) (*StreamMarkersRe
 
 	return markers, nil
 }
+
+// CreateStreamMarker ...
+type CreateStreamMarker struct {
+	ID              string `json:"id"`
+	CreatedAt       Time   `json:"created_at"`
+	Description     string `json:"description"`
+	PositionSeconds int    `json:"position_seconds"`
+}
+
+// ManyCreateStreamMarkers ...
+type ManyCreateStreamMarkers struct {
+	CreateStreamMarkers []CreateStreamMarker `json:"data"`
+}
+
+// CreateStreamMarkerResponse ...
+type CreateStreamMarkerResponse struct {
+	ResponseCommon
+	Data ManyCreateStreamMarkers
+}
+
+// CreateStreamMarkerParams ...
+type CreateStreamMarkerParams struct {
+	UserID      string `query:"user_id"`
+	Description string `query:"description"`
+}
+
+// CreateStreamMarker creates a stream marker for a live stream at the current time.
+// The user has to be the stream owner or an editor. Stream markers cannot be created
+// in some cases, see:
+// https://dev.twitch.tv/docs/api/reference/#create-stream-marker
+//
+// Required Scope: user:edit:broadcast
+func (c *Client) CreateStreamMarker(userID, description string) (*CreateStreamMarkerResponse, error) {
```

Comment:
I'd prefer we pass `CreateStreamMarkerParams` as the param argument here rather than `userID` & `description`. Two reasons for this:

- `description` is optional. Also I'm not a huge fan of using empty strings for optional arguments; and
- Twitch is notorious for adding additional params to their requests (see #10), as the Helix API is continually being improved by their developers